docs/hopsworksai/rondb.md

```diff
@@ -0,0 +1,71 @@
+# Managed RonDB
+For applications where Feature Store's performance and scalability is paramount we give users the option to create clusters with
+Managed [RonDB](https://www.rondb.com/). You don't need to worry about configuration as [hopsworks.ai](https://managed.hopsworks.ai/) will
+automatically pick the best options for your setup.
+
+## Enabling RonDB
+To setup a cluster with RonDB, click on the `Enable` checkbox during cluster creation. If you don't see this option contact [sales](mailto: sales@logicalclocks.com).
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/rondb/rondb_enable.png">
+      <img style="border: 1px solid #000" src="../../../assets/images/hopsworksai/rondb/rondb_enable.png" alt="Enable Managed RonDB">
+    </a>
+    <figcaption>Enable Managed RonDB</figcaption>
+  </figure>
+</p>
+
+## Node selection
+If you enable Managed RonDB you will see a basic configuration page where you can select the number of database nodes.
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/rondb/rondb_basic.png">
+      <img style="border: 1px solid #000" src="../../../assets/images/hopsworksai/rondb/rondb_basic.png" alt="RonDB basic configuration">
+    </a>
+    <figcaption>RonDB basic configuration</figcaption>
+  </figure>
+</p>
+
+First, you need to select the number of *Data nodes*. These are the database nodes that will store data. The number
+of Datanodes is dependent on the [number of replicas](#number-of-replicas). For most applications 2 Datanodes will suffice with
+memory optimized instance type. Local storage is not very important as RonDB is an in-memory database but it has to be
+big enough for offline storage of recovery data.
+
+Next you can configure the number of MySQLd nodes. If you have a specific use-case where you expect high load on MySQL servers,
```

Comment:
Could we give an example of such a use-case, most user will not know what use the MySQL servers.